src/app/components/ATIAnalytics/types.ts

```diff
@@ -161,6 +161,21 @@ export interface ATIEventTrackingProps {
   experimentVariant?: string;
   ampExperimentName?: string;
   preventNavigation?: string;
+  componentSpecificTrackers?: ComponentSpecificTrackers;
+  blockSpecificTrackers?: BlockSpecificTrackers;
+}
+
+export interface ComponentSpecificTrackers {
+  type?: string;
+  text?: string;
+  position?: number;
+  duration?: number;
+  resourceId?: string;
+}
+
+export interface BlockSpecificTrackers {
```

Comment:
Instead of a block, could we use the word Group (as that's the part of the viewability model that's used?) And as above, let's not use the word `Specific`

```suggestion
export interface GroupTracking {
```

```suggestion
export interface GroupTracker {
```